docs/content/technical-specifications.md

```diff
@@ -19,6 +19,19 @@ The 1.12 release supports the Ingress v1beta1 API and continues to receive secur
 
 We explicitly test the NGINX Ingress Controller on a range of Kubernetes platforms at each release, and the [release notes](/nginx-ingress-controller/releases) list which platforms were tested. We will provide technical support for the NGINX Ingress Controller on any Kubernetes platform that is currently supported by its provider and which passes the [Kubernetes conformance tests](https://www.cncf.io/certification/software-conformance/).
 
+{{% table %}}
+| NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
+| --- | --- | --- | --- | --- |
+| 2.1.1 | 1.23 - 1.19 | 0.12.0 |  | 1.21.5 / R25 |
```

Comment:
per this doc https://github.com/nginxinc/nginx-ingress-operator#nginx-ingress-operator we do have operator for 2.1.x and 2.0.x versions of NIC